src/AuthenticatedAPIClient/authInterface.js

```diff
@@ -83,8 +71,11 @@ export default function applyAuthInterface(httpClient, authConfig) {
     global.location.assign(`${httpClient.loginUrl}?next=${encodeURIComponent(redirectUrl)}`);
   };
 
-  httpClient.logout = (redirectUrl = authConfig.appBaseUrl) => {
-    global.location.assign(`${httpClient.logoutUrl}?redirect_url=${encodeURIComponent(redirectUrl)}`);
+  httpClient.logout = (redirectUrl = authConfig.appBaseUrl, errorCode = null) => {
+    const errorCodeParam = errorCode ? `&error_code=${errorCode}` : '';
```

Comment:
I suppose we're assuming that `errorCode` is already URL encoded here.

Reply:
Removed change.  Thanks.